src/errors.rs

```diff
@@ -219,6 +222,7 @@ pub fn http_status_code_for_service_error(error: &ServiceError) -> StatusCode {
         ServiceError::InvalidTorrentFile => StatusCode::BAD_REQUEST,
         ServiceError::InvalidTorrentPiecesLength => StatusCode::BAD_REQUEST,
         ServiceError::InvalidFileType => StatusCode::BAD_REQUEST,
+        &ServiceError::InvalidTorrentTitleLength => StatusCode::BAD_REQUEST,
```

Comment:
Hi @alexohneander why are you referencing the reference here?

https://github.com/rust-lang/rfcs/blob/master/text/2005-match-ergonomics.md

Reply:
@josecelano Here you are absolutely right, I would fix this and recreate the pull request.